js/whisper-demo/main.js

```diff
@@ -345,10 +360,14 @@ async function processAudioBuffer() {
                 recognition._onresult(ret, true);
             }
```

Comment:
If it is `accumulateSubChunks` mode, `subText` should be refreshed every time until the `subAudioChunks` is empty.